guestbook/README.md

```diff
@@ -4,704 +4,354 @@
 > https://kubernetes.io/docs/tutorials/stateless-application/guestbook/.
 > Otherwise some of the URLs will not work properly.
 
-## Guestbook Example
+## Creating a Guestbook
 <!-- EXCLUDE_FROM_DOCS END -->
 
-This example shows how to build a simple, multi-tier web application using Kubernetes and [Docker](https://www.docker.com/).
+{% capture overview %}
+This tutorial shows you how to build a simple, multi-tier web application using Kubernetes and [Docker](https://www.docker.com/). This example consists of the following:
 
-**Table of Contents**
-<!-- BEGIN MUNGE: GENERATED_TOC -->
+* a web frontend 
+* a [Redis](https://redis.io/) master for storage 
+* a replicated set of Redis slaves. 
```

Comment:
multiple replicated redis instances
OR
multiple replicated redis instances, used to serve reads

I think "replicated set" is capturing the fact that this was once a ReplicaSet. but not antymore.